Tools/gen_manifest

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python
+
+import os
+import sys
+import subprocess
+import re
+from shutil import copyfile
+runtime_libs = []
+def init_runtime_libs () :
+  runtime_libs.append('libc')
+  runtime_libs.append('libdl')
+  runtime_libs.append('libm')
+  runtime_libs.append('libpthread')
+  runtime_libs.append('libutil')
+  runtime_libs.append('libnss_dns')
+  runtime_libs.append('libresolv')
+  runtime_libs.append('librt')
+  
+def filter_default_libs (lib_name) :
+  res = True
+  for lib in runtime_libs :
+    if lib_name == lib :
+      res = False
+  return res
```

Comment:
This whole function can be simplified to:
```Python
def filter_default_libs(lib_name):
    return lib_name not in runtime_libs
```

Reply:
IMO this function could be removed entirely, using `lib_name not in runtime_libs` instead of it seems more readable to me.

Reply:
Looks good now, thanks.